apps/apps.go

```diff
@@ -56,14 +58,18 @@ func New(c *deis.Client, id string) (api.App, error) {
 		}
 	}
 
-	resBody, err := c.BasicRequest("POST", "/v2/apps/", body)
-
+	res, err := c.Request("POST", "/v2/apps/", body)
 	if err != nil {
 		return api.App{}, err
 	}
+	// Fix json.Decoder bug in <go1.7
```

Comment:
It was caught in google's github-api repo: https://github.com/google/go-github/pull/317

I'll make a PR to reformat the warning to what you suggested everywhere.